operator/Makefile

```diff
@@ -187,15 +187,12 @@ olm-deploy:  ## Deploy the operator bundle and the operator via OLM into an Kube
 	$(error Set variable REGISTRY_ORG to use a custom container registry org account for local development)
 else
 olm-deploy: olm-deploy-bundle olm-deploy-operator $(OPERATOR_SDK)
-	kubectl create ns $(LOKI_OPERATOR_NS)
-	kubectl label ns/$(LOKI_OPERATOR_NS) openshift.io/cluster-monitoring=true --overwrite
 	$(OPERATOR_SDK) run bundle -n $(LOKI_OPERATOR_NS) --install-mode OwnNamespace $(BUNDLE_IMG)
```

Comment:
One thing I noticed during testing last week is that the operator-sdk will complain about the install-mode once there's another operator in that namespace (for example, the elasticsearch-operator). Do we want to fix this in another PR or does it make sense to lump it in here?

Reply:
Good catch, let's fix it here to address this in addition to all the other shortcomings introduced by #5440 .